packages/components/src/internal/app/constants.ts

```diff
@@ -3,16 +3,23 @@
  * any form or by any electronic or mechanical means without written permission from LabKey Corporation.
  */
 import { AppURL } from '../url/AppURL';
+import { AppProperties } from './models';
+import { imageURL } from '../url/ActionURL';
 
 // These ids should match what is used by the MenuProviders in the Java code so we can avoid toLowerCase comparisons.
 export const LKS_PRODUCT_ID = 'LabKeyServer';
-export const BIOLOGICS_PRODUCT_ID = 'Biologics';
-export const SAMPLE_MANAGER_PRODUCT_ID = 'SampleManager';
-export const FREEZER_MANAGER_PRODUCT_ID = 'freezerManager';
+const BIOLOGICS_PRODUCT_ID = 'Biologics';
+const SAMPLE_MANAGER_PRODUCT_ID = 'SampleManager';
+const FREEZER_MANAGER_PRODUCT_ID = 'freezerManager';
```

Comment:
minor, but is there a reason that this FM product ID has a different casing then the others?